components/mediation-ui/mediators-ui/org.wso2.carbon.mediator.store.ui/src/main/java/org/wso2/carbon/mediator/store/MessageStoreMediator.java

```diff
@@ -135,4 +161,15 @@ private void handleException(String msg) {
         LogFactory.getLog(this.getClass()).error(msg);
         throw new SynapseException(msg);
     }
+
+    public boolean hasBraces(OMAttribute atr) {

```

Comment:
This could be private as no use outside of the class.

Reply:
Fixed with https://github.com/wso2/carbon-mediation/pull/1012